styx-scheduler-service/src/main/java/com/spotify/styx/state/PersistentStateManager.java

```diff
@@ -134,12 +134,18 @@ public void tick() {
   private void tickInstance(WorkflowInstance instance) {
     try {
       var stateOpt = storage.readActiveState(instance);
-      stateOpt.ifPresent(state -> {
-        LOG.info("Ticking instance: {}: #{} {}", instance, state.counter(), state.state());
-        outputHandler.transitionInto(state);
-      });
+      stateOpt.ifPresent(state -> tickInstance(instance, state));
     } catch (Exception e) {
-      LOG.error("Error ticking instance: {}", instance, e);
+      log.error("Error ticking instance: {}", instance, e);
+    }
+  }
+
+  private void tickInstance(WorkflowInstance instance, RunState state) {
```

Comment:
`instance` is part of `RunState`, so we don't need that argument.

Reply:
👍